pkg/ebpf/c/common/context.h

```diff
@@ -160,16 +170,21 @@ statfunc int init_program_data(program_data_t *p, void *ctx)
 
     bool container_lookup_required = true;
 
-    p->task_info = bpf_map_lookup_elem(&task_info_map, &p->event->context.task.host_tid);
+    u32 host_pid = p->event->context.task.host_pid;
+    p->proc_info = bpf_map_lookup_elem(&proc_info_map, &host_pid);
+    if (unlikely(p->proc_info == NULL)) {
+        p->proc_info = init_proc_info(host_pid, p->scratch_idx);
```

Comment:
As I didn't detect the initialisation of `p->scratch_idx` can we assume that it is always 0 here, the default value?

Reply:
Yes, we can, since we initialise p as `program_info_t p = {};` in all programs.